cmd/sks_list.go

```diff
@@ -65,18 +65,18 @@ func listSKSClusters(zone string) outputter {
 			return fmt.Errorf("unable to list SKS clusters in zone %s: %v", zone, err)
 		}
 
-		for _, nlb := range list {
+		for _, cluster := range list {
 			res <- sksClusterListItemOutput{
-				ID:   nlb.ID,
-				Name: nlb.Name,
+				ID:   cluster.ID,
+				Name: cluster.Name,
 				Zone: zone,
 			}
 		}
 
 		return nil
 	})
 	if err != nil {
-		fmt.Fprintf(os.Stderr,
+		_, _ = fmt.Fprintf(os.Stderr,
```

Comment:
Out of curiosity, why do we have to capture to dead variables the output of `fmt.Fprintf`?

Reply:
Otherwise my overzealous IDE complains that the returned error is not checked 😒